Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.mm

```diff
@@ -1044,14 +1065,17 @@ - (BOOL)prefersStatusBarHidden
     if (m_mockVideoPresentationModeEnabled)
         return;
 
-    if (auto interface = findInterface(contextId))
+    if (auto interface = findInterface(contextId)) {
         interface->hasVideoChanged(hasVideo);
+        ensureModel(contextId)->m_hasVideo = hasVideo;
+    }
```

Comment:
We should probably add something like `findInterfaceAndModel(...)` so functions like this don't have to search the hash map twice.